docs/tutorials/excel-power-automate-manual.md

```diff
@@ -0,0 +1,118 @@
+- Ask your administrator to [enable Office Scripts for your organization](https://support.office.com/article/office-scripts-settings-in-m365-19d3c51a-6ca2-40ab-978d-60fa49554dcf), which adds the **Automate** tab to the ribbon in Excel on the web.
+- [Preview access to Power Automate](https://us.tip1.flow.microsoft.com).
+
+> [!IMPORTANT]
+> This tutorial assumes you have completed the [Record, edit, and create Office Scripts in Excel on the web](excel-tutorial.md) tutorial.
+
+## Prepare the workbook
+
+Power Automate can't use relative references like `Workbook.getActiveWorksheet` to access workbook components. So, we need a workbook and worksheet with consistent names that Power Automate can reference.
+
+1. Create a new workbook named **MyWorkbook**.
+
+2. In the **MyWorkbook** workbook, create a worksheet called **TutorialWorksheet**.
+
+## Create an Office Script
+
+1. Go to the **Automate** tab and select **Code Editor**.
+
+2. Select **New Script**.
+
+3. Replace the default script with the following script. This script adds the current date and time to the first two cells of the **TutorialWorksheet** worksheet.
+
+    ```TypeScript
+    async function main(context: Excel.RequestContext) {
+      // Get the "TutorialWorksheet" worksheet from the workbook.
+      let worksheet = context.workbook.worksheets.getItem("TutorialWorksheet")
+
+      // Get the cells at A1 and B1.
+      let dateRange = worksheet.getRange("A1");
+      let timeRange = worksheet.getRange("B1");
+
+      // Get the current date and time using the JavaScript Date object.
+      let date = new Date(Date.now());
+
+      // Add the date string to A1.
+      dateRange.values = [[date.toLocaleDateString()]];
+
+      // Add the time string to B1.
+      timeRange.values = [[date.toLocaleTimeString()]];
+    }
+    ```
+
+4. Rename the script to **Set date and time**. Press the script name to change it.
+
+5. Save the script by pressing **Save Script**.
+
+## Create an automated workflow with Power Automate
+
+1. Sign in to [Power Automate](https://us.tip1.flow.microsoft.com).
+
+2. In the menu that's displayed on the left side of the screen, press **Create**. This brings you to list of ways to create new workflows.
+
+    ![The Create button in Power Automate.](../images/power-automate-tutorial-1.png)
+
+3. In the **Start from blank** section, select **Instant flow**. This creates a manually activated workflow.
+
+    ![The Instant flow option for creating a new workflow.](../images/power-automate-tutorial-2.png)
+
+4. In the dialog window that appears, enter a name for your flow in the **Flow name** text box, select **Manually trigger a flow** from the list of options under **Choose how to trigger the flow**, and press **Create**.
+
+    ![The manual trigger option for creating a new instant flow.](../images/power-automate-tutorial-3.png)
+
+5. Press **New step**.
+
+6. Select the **Custom** tab. Under **Actions**, select **Run script (preview)**.
```

Comment:
This should actually say the main Excel Online (Business) connector—_not_ the custom connector. It also might be helpful to have screenshots for steps (5) and (6).

Reply:
@danielgwilson, I'll make the change, though the Run Script action is not yet present. I'll add the screenshots once the option is live.